plugin/dependency-i18n-locale.js

```diff
@@ -33,10 +33,11 @@ var types = {
 
         var moduleName = compilerContext.dictionaryModuleName + '/' + compilerContext.key + '.marko';
 
-        markoCompiler.compile('---\n' + value + '\n---', templatePath, function(err, code) {
+        markoCompiler.compileForBrowser('---\n' + value + '\n---', templatePath, function(err, compiledTemplate) {
```

Comment:
What do you think about making this backwards compatible with Marko v3? Otherwise this module will require a full version change since Marko v3 does not have the `compileForBrowser` function.